src/components/molecules/CameraMicrophoneControls/CameraMicrophoneControls.tsx

```diff
@@ -21,37 +37,16 @@ export const CameraMicrophoneControls: React.FC<CameraMicrophoneControlsProps> =
   defaultMute,
   containerClassName,
 }) => {
-  const getChangeStateHandler = useCallback(
-    (variant: "microphone" | "camera"): ((enabled: boolean) => void) => {
-      const tracks =
-        variant === "microphone"
-          ? participant.audioTracks
-          : participant.videoTracks;
-
-      return (enable: boolean) => {
-        tracks.forEach(
-          (track: AudioTrackPublication | VideoTrackPublication) => {
-            const innerTrack = track.track;
-            if (innerTrack) {
-              if (enable && "enable" in innerTrack) innerTrack.enable();
-              else if ("disable" in innerTrack) innerTrack.disable();
-            }
-          }
-        );
-      };
-    },
-    [participant.audioTracks, participant.videoTracks]
+  const changeAudioState = useCallback(
+    (state) => participant.audioTracks.forEach(setTrackStateTo(state)),
+    [participant.audioTracks]
   );
```

Comment:
No `eslint-disable` comments.
One line explaining what's being done to each audio track, one line for the deps.

Too bad twilio uses Map with its clunky interface... `forEach` wins out over `for-of` this time